samples/wormhole/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: NServiceBus.WormHole Samples
```

Comment:
No change to "Wormhole"? Is that a problem because it would be a breaking change?